app/views/catarse_bootstrap/user_notifier/mailer/sorbonne.html.slim

```diff
@@ -0,0 +1,46 @@
+p
+  | Caro #{@notification.contribution.user.display_name},
+p
+  br/
+  br/
+p
+  | O Catarse está colaborando em um estudo da Universidade de Paris (Sorbonne). Eles querem investigar as personalidades e as motivações das pessoas que apóiam projetos de financiamento coletivo.
+p
+  br/
+  | Você está em um grupo de apoiadores de projetos na plataforma selecionados por eles para responder a uma pesquisa e ajudar no desenvolvimento de um trabalho científico para compreendermos melhor esse modelo que já ajudou milhares de ideias a saírem do papel no Brasil e no mundo.
+p
+  br/
+  u> Todos os participantes serão remunerados com um valor entre R$15 e R$90
+  | dependendo do seu desempenho, e todas as respostas serão anônimas. O estudo e o pagamento são de responsabilidade Université de Paris.
+p
+  | Algumas questões práticas:
+ul
+  li
+    | Caso tenha alguma dúvida, você poderá entrar em contato direto com os pesquisadores através do e-mail:
+    a href="mailto:pesquisa@univ-paris.fr"  pesquisa@univ-paris.fr
+    | (o contato pode ser feito em português).
+  li O tempo previsto para responder a pesquisa é entre 40 minutos e 1 hora.
+  li
+    ' Recomenda-se a utilização da versão
+    u> atualizada
+    | dos navegadores Google Chrome, Firefox ou Safari.
+  li O número de vagas disponíveis para esta sessão do estudo são limitados, será dada preferência para os primeiros participantes.
+  li Todos pagamentos para os participantes da pesquisa serão efetuados via PayPall no dia 25/03/2015, independente de quando você participar da pesquisa.
+p
+  'Para obter mais informações e responder a pesquisa, acesse: 
+  = link_to 'pesquisa.fr', 'pesquisa.fr'
+p
+  | Seu login : [LOGIN]
+p
+  br/
+  br/
+p
+  | Obrigado!
+p
+  br/
+  br/
+p
+  | Equipe do Catarse
+
+= image_tag 'logo_sorbonne.jpg', height: '181', width: '400'
```

Comment:
For the email we need the full URL of the image. I believe that the image_tag will generate a relative path no?